modules/tinymce/src/themes/silver/main/ts/ui/sidebar/Sidebar.ts

```diff
@@ -94,9 +94,20 @@ const makeSidebar = (panelConfigs: SidebarConfig) => SlotContainer.sketch((parts
   ])
 }));
 
-const setSidebar = (sidebar: AlloyComponent, panelConfigs: SidebarConfig) => {
+const setSidebar = (sidebar: AlloyComponent, panelConfigs: SidebarConfig, showSidebar?: string) => {
```

Comment:
This showSidebar should either be an option or a `string | undefined` type since having a ? on it makes it an optional parameter and I think that is not the intent here.